.github/workflows/sync-labels.yml

```diff
@@ -0,0 +1,21 @@
+name: Tools
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - .github/labels.yml
+      - .github/workflows/sync-labels.yml
+  schedule:
+    - cron: 6 6 6 * *
```

Comment:
@ErikSchierboom 👀 that was supposed to be `0 0 1`

Reply:
That's weird!

Reply:
Found the bug. Fixing it for all involved repos now (basically all repos which name is > `org-wide-files` :))